.github/workflows/e2e_testnet_daily.yml

```diff
@@ -56,37 +117,56 @@ jobs:
       contents: read
     name: Check If Solana Test Image Exists
     runs-on: ubuntu-latest
+    needs: [ get_solana_sha ]
     outputs:
       exists: ${{ steps.check-image.outputs.exists }}
     steps:
       - name: Check if image exists
         id: check-image
-        uses: smartcontractkit/chainlink-github-actions/docker/image-exists@ad22fbd6f4d108b82aaf49b527bcf40f32babea8 # v2.2.1
+        uses: smartcontractkit/chainlink-github-actions/docker/image-exists@e865e376b8c2d594028c8d645dd6c47169b72974 # v2.2.16
         with:
           repository: chainlink-solana-tests
```

Comment:
What is this repository ID?

Reply:
Internal Chainlink action to check fi image has been build already on this commit. It is used everywhere in tests